src/table/virtualized-table/HeaderCell.tsx

```diff
@@ -17,8 +18,10 @@ const HeaderCell = ({ index, style, data }: HeaderCellProps) => {
     columns,
     headerStyle: { ...applicableStyle },
   } = data;
+  const { layout } = useContextSelector(TableContext, (value) => value.baseProps);
   const datum = columns[index];
```

Comment:
Might as well rename `datum` to `column`

Reply:
agree